views/form.html

```diff
@@ -40,20 +40,26 @@ <h2 class="prospect-form__heading">And get 30 more days of free FT</h2>
 			{{#unless isUnmasking}}
 			{{> n-conversion-forms/partials/email isB2b=true }}
 			{{/unless}}
-			
+
 			{{> n-conversion-forms/partials/phone isB2b=true }}
 
 			{{#unless isUnmasking}}
 			{{> n-conversion-forms/partials/country options=countries }}
 			{{/unless}}
 
-			{{#> n-conversion-forms/partials/fieldset name="accept-terms" legend="Terms and conditions" }}
-			{{#*inline "fields"}}
+			{{#ifAll @root.flags.channelsBarrierConsent consent}}
+				<div>
```

Comment:
Why the random div?

Reply:
No particular reason, just wanted a single container element (the partial doesn't render one).